kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/SchemaRegistryService.java

```diff
@@ -57,11 +68,19 @@
   private final ClusterMapper mapper;
   private final WebClient webClient;
 
-  public Flux<SchemaSubjectDTO> getAllLatestVersionSchemas(KafkaCluster cluster) {
-    var allSubjectNames = getAllSubjectNames(cluster);
-    return allSubjectNames
-        .flatMapMany(Flux::fromArray)
-        .flatMap(subject -> getLatestSchemaVersionBySubject(cluster, subject));
+  public Mono<SchemaSubjectsResponseDTO> getAllLatestVersionSchemas(KafkaCluster cluster,
+                                                                    Optional<Integer> pageNum,
```

Comment:
I think it s better to filter and fill parameters on controller level and pass validated params to service (I know that you were inspired by TopicsService, but nevertheless)